src/mods/depositAll/helper.js

```diff
@@ -0,0 +1,24 @@
+function deposit() {
+	// Select normal deposit button
+	document
+		.querySelector('.slot .grey.gold:not(.js-deposit-all)')
+		.dispatchEvent(new Event('click'));
+
+	// Input some huge value they'll have less than
+	document.querySelector('input.formatted').value = 999999999999999;
```

Comment:
Can we only call `document.querySelector` the minimum times possible? Pretty sure it makes calls to the DOM each time we invoke the method.

Same feedback throughout the PR.

For example,
```
const $currencyInput = $stash.querySelector('input.formatted');
$currencyInput.value = 99999999;
$currencyInput.dispatchEvent...
```

Reply:
Reduced calls to querySelector where possible